<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlaybackClass</class>
 <widget class="QWidget" name="VideoPlaybackClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoPlayback</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="8,1,1">
   <item>
    <widget class="OpenGLPreviewWidget" name="openGLWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="MySlider" name="videoTImeSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeCodeLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="startPushButton">
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pausePushButton">
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="continuePushButton">
         <property name="text">
          <string>继续</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="choosePushButton">
         <property name="text">
          <string>选择文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fileNameLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OpenGLPreviewWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>ui/OpenGLPreviewWidget/OpenGLPreviewWidget.h</header>
  </customwidget>
  <customwidget>
   <class>MySlider</class>
   <extends>QSlider</extends>
   <header>MySlider/MySlider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../VideoPlayback.qrc"/>
 </resources>
 <connections/>
</ui>
